packages/core/admin/admin/src/content-manager/history/pages/History.tsx

```diff
@@ -80,7 +87,10 @@ const HistoryPage = () => {
         )}
       />
       <Flex direction="row" alignItems="flex-start">
-        <VersionDetails version={selectedVersion} />
+        <Main grow={1} labelledBy={headerId}>
+          <VersionHeader version={selectedVersion} layout={layout} headerId={headerId} />
```

Comment:
could just pass the contentType or other partial data instead of the entire layout 🤔